runtime/common/CMakeLists.txt

```diff
@@ -86,19 +143,14 @@ endif()
 
 add_library(cudaq-mlir-runtime
   SHARED
-    ArgumentConversion.cpp
     Environment.cpp
-    JIT.cpp
     Logger.cpp
```

Comment:
Seeing Logger.cpp and Environment.cpp here for the first time - these are part of common, I wonder why they are here too? Shouldn't be. 

With that, then you don't even have a cudaq-mlir-runtime library any more. You would just have the cudaq-mlir-core and the cudaq-mlir-compiler.